admin/tests/integration/components/confirm-popup-test.js

```diff
@@ -1,22 +1,83 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render } from '@ember/test-helpers';
+import { click, render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
 
 module('Integration | Component | confirm-popup', function(hooks) {
   setupRenderingTest(hooks);
 
-  test('it renders', async function(assert) {
-    // Set any properties with this.set('myProperty', 'value');
-    // Handle any actions with this.set('myAction', function(val) { ... });
-    this.set('display', true);
-    this.set('actionConfirm', () => {
+  module('when display is true', function(hooks) {
```

Comment:
Exprimer l'intention fonctionnellement d'un point de vue d'utilisateur ?

Reply:
`this.display = true` est un prérequis.
Il a sa place dans un before.

Reply:
J'ai retiré le module (il n'y en avait pas d'autres de toute façon) et j'ai laissé le `beforeEach`